app/models/agreement_letter.rb

```diff
@@ -15,7 +15,7 @@ class AgreementLetter < ActiveRecord::Base
   belongs_to :event
   validates_presence_of :user, :event, :path
   MAX_SIZE = 300_000_000
-  ALLOWED_MIMETYPE = "application/pdf"
+  ALLOWED_MIMETYPE = 'application/pdf'.freeze
```

Comment:
[`freeze`](https://apidock.com/ruby/Object/freeze) prevents further modification of the object. A RuntimeError will be raised if modification is attempted.